querier.go

```diff
@@ -876,6 +879,10 @@ func (s *chunkSeries) Iterator() SeriesIterator {
 	return newChunkSeriesIterator(s.chunks, s.intervals, s.mint, s.maxt)
 }
 
+func (s *chunkSeries) Chunks() []chunks.Meta {
+	return s.chunks
```

Comment:
Is that a copy of chunks as comment suggests? I think it is exactly the same underlying array passed around.